precommit/src/main/shell/plugins.d/github.sh

```diff
@@ -1055,6 +1059,11 @@ function github_finalreport
     return 0
   fi
 
+  if [[ "${GITHUB_WRITE_COMMENT}" == true ]]; then
+    github_finalreport_as_comment result
+    return 0
```

Comment:
You cannot do a return here as it cuts out the rest of the code.

Reply:
Fixed https://github.com/apache/yetus/pull/213/commits/18bde22af1134f42ba0afa87755b72f9516ebfbd